### ISSUE TITLE
CLI tool exits with 0 status even with fatal errors

### ISSUE THREAD

**Issue by 1ec5** [mozilla/addons-linter#417]
Even when there are fatal errors, this tool exits with a 0 status. I’d like to run addons-linter in my add-on’s continuous integration script, but errors won’t cause the build to fail. Instead, I have to parse the JSON output or grep it for `"count":0`.

**Comment by 1ec5** [comment]
I attempted to run addons-linter on the [addons-linter](https://github.com/1ec5/avim/compare/addons-linter) branch. However, Travis just doesn’t have a good environment for it right now. addons-linter needs node 0.12+, whereas Travis’ Python containers come with node 0.10. If we switch to a Node.js container, `pip` doesn’t work because it needs `sudo`. Using `sudo` is a non-starter, because it means we can’t use the container-based infrastructure, which allows for much quicker builds as well as caching of the SpiderMonkey dependency. So while the branch works, I’m unlikely to merge it.

I also had to work around mozilla/addons-linter#417 by running the linter twice, the second time grepping JSON output.
